std/process.d

```diff
@@ -1095,14 +1101,16 @@ private Pid spawnProcessPosix(scope const(char[])[] args,
                 if (stderrFD > STDERR_FILENO)  close(stderrFD);
             }
 
+            config.preExecFunction(&abortOnErrorInPreExec);
```

Comment:
Why not simply make `preExecFunction` return a `bool`?

Reply:
It is better if `preExecFunction` is nothrow.
I fixed a signature and removed unused `abortOnErrorInPreExec`.